src/Microsoft.Health.SqlServer/Features/Schema/SchemaInformation.cs

```diff
@@ -18,5 +18,14 @@ public SchemaInformation(int minimumSupportedVersion, int maximumSupportedVersio
         public int MaximumSupportedVersion { get; }
 
         public int? Current { get; set; }
+
+        /// <summary>
+        /// Checks if the version is not inside the full schema script.
```

Comment:
It might be good to add a bit more context here about how previous full snapshot schemas set the schema version and that we are looking to prevent future SQL schemas from having to do so.